.gitignore

```diff
@@ -1,14 +1,11 @@
 # Hugo-generated assets
-public/
+/public
 resources/
 
 # npm assets
 node_modules/
 
-# ignore themes
-themes/
```

Comment:
Unless I'm misunderstanding the intent here, we don't want to ignore the top-level `themes` directory -- it was committed and is being tracked already. I doubt that the intent was to ignore `themes` subdirectories.